config/systemd/omnect-ui.env.template

```diff
@@ -5,3 +5,5 @@ LOGIN_USER="%%USER%%"
 LOGIN_PASSWORD="%%PASSWORD%%"
 RUST_LOG="info"
 UI_PORT="1977"
+CERT_PATH=$(echo /var/lib/aziot/certd/certs/deviceid-*.cer),target=/cert/device_id_cert.pem)
```

Comment:
does that work?
parentheses are unbalanced: there's a trailing ')'.
also, this might also fail in case several files match on the pattern, although I guess that this won't happen in the wild.
(no file existing which matches the pattern would probably also fail, though)
and, yes, I know that all but the parentheses mismatch is not new in this change, just wanted to mention it anyway ;-)

Reply:
thx.  I didn't test yet